util/dftool_module.py

```diff
@@ -19,8 +19,9 @@
     networkutil,
     query,
 )
+from util.predictoor.query import queryPredictoors
```

Comment:
Remove. The import to `query` above is enough. Then below, just use it as `query.queryPredictors`. 

Why: the way it's currently written, now you have to write `queryPredictoors` twice. Harder to maintain 2 copies of every query function name in this module, versus a single `import query`. Calina and I discussed. At the very least, we need to maintain consistency with the current way.

Reply:
I would like to mention that we discussed, but I lean towards how Berkay did it here. We did not reach any conclusion. However, we can keep consistency for now and "argue" later :)

Reply:
In this case I guess we can't use
from util.predictoor import query 

Because that would be a different query module 

So I guess leave it as-is here

Reply:
I agree with @trizin. That's and advantage of the imports with "from", which allows modularisation. With most frameworks and Python projects it is common to have files with the same name in different modules. If you import those functions mixed in a single entry point, using "from..." is much straightforward than aliasing

Reply:
OK let's use "from" here and other places where we have >1  module of the same name yet different directory. We agree for sure on that.

For the rest: rather than saying always use one way or another, can we agree to whatever gives the lowest complexity? If a case is about the same for one way vs the other, then I'm fine with either.